datadog_checks_dev/datadog_checks/dev/conditions.py

```diff
@@ -14,6 +15,8 @@
 from .subprocess import run_command
 from .utils import file_exists
 
+logger = logging.getLogger(__name__)
```

Comment:
Please use `echo_debug`, example here: https://github.com/Datadog/integrations-core/blob/28f3946b61bc0d87ad5b55b8c0b4e1e5161e07dd/datadog_checks_dev/datadog_checks/dev/tooling/testing.py#L61

Reply:
It looks like we can't import `echo_debug` from the `datadog_checks/dev` level due to a circular import:

```
Traceback (most recent call last):
  File "/usr/local/anaconda3/envs/agent38/bin/ddev", line 11, in <module>
    load_entry_point('datadog-checks-dev', 'console_scripts', 'ddev')()
  File "/usr/local/anaconda3/envs/agent38/lib/python3.8/site-packages/pkg_resources/__init__.py", line 490, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "/usr/local/anaconda3/envs/agent38/lib/python3.8/site-packages/pkg_resources/__init__.py", line 2862, in load_entry_point
    return ep.load()
  File "/usr/local/anaconda3/envs/agent38/lib/python3.8/site-packages/pkg_resources/__init__.py", line 2453, in load
    return self.resolve()
  File "/usr/local/anaconda3/envs/agent38/lib/python3.8/site-packages/pkg_resources/__init__.py", line 2459, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
  File "/Users/dusan.jovanovic/go/src/github.com/DataDog/integrations-core/datadog_checks_dev/datadog_checks/dev/__init__.py", line 5, in <module>
    from .conditions import WaitFor
  File "/Users/dusan.jovanovic/go/src/github.com/DataDog/integrations-core/datadog_checks_dev/datadog_checks/dev/conditions.py", line 17, in <module>
    from .tooling.commands.console import echo_debug
  File "/Users/dusan.jovanovic/go/src/github.com/DataDog/integrations-core/datadog_checks_dev/datadog_checks/dev/tooling/commands/__init__.py", line 10, in <module>
    from .docs import docs
  File "/Users/dusan.jovanovic/go/src/github.com/DataDog/integrations-core/datadog_checks_dev/datadog_checks/dev/tooling/commands/docs/__init__.py", line 7, in <module>
    from .build import build
  File "/Users/dusan.jovanovic/go/src/github.com/DataDog/integrations-core/datadog_checks_dev/datadog_checks/dev/tooling/commands/docs/build.py", line 8, in <module>
    from .... import chdir
ImportError: cannot import name 'chdir' from partially initialized module 'datadog_checks.dev' (most likely due to a circular import) (/Users/dusan.jovanovic/go/src/github.com/DataDog/integrations-core/datadog_checks_dev/datadog_checks/dev/__init__.py)
```

Anything wrong with using a logger? It seems to work OK. Here's what I see with the logger:

```
DEBUG    datadog_checks.dev.conditions:conditions.py:40 WaitFor error func=connect_to_pg
Traceback (most recent call last):
  File "/Users/dusan.jovanovic/go/src/github.com/DataDog/integrations-core/datadog_checks_dev/datadog_checks/dev/conditions.py", line 38, in __call__
    result = self.func(*self.args, **self.kwargs)
  File "/Users/dusan.jovanovic/go/src/github.com/DataDog/integrations-core/postgres/tests/conftest.py", line 23, in connect_to_pg
    psycopg2.connect(host=HOST, dbname=DB_NAME, user=USER, password=PASSWORD)
  File "/Users/dusan.jovanovic/go/src/github.com/DataDog/integrations-core/postgres/.tox/py27/lib/python2.7/site-packages/psycopg2/__init__.py", line 126, in connect
    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)
OperationalError: server closed the connection unexpectedly
	This probably means the server terminated abnormally
	before or while processing the request.
```